### ISSUE TITLE
Is this project stopped?

### ISSUE THREAD

**Issue by win32nipuh** [saaskit/saaskit#105]
Does it work with NET Core 3.1 ?
Blazor?

**Comment by gordon-matt** [comment]
Yeah, no changes in 9 years.. I think it's fair to say it's dead. That said, it still works.. I am using it in .NET9 with no issues. I've just submitted a PR for upgrading everything to .NET Standard 2, but I don't have much hope it will actually even be viewed, much less approved. After submitting the PR, I noticed others did similar work a while back as well.
